manifests/o/OpenStreetMap/Osm2pgsql/1.7.2/OpenStreetMap.Osm2pgsql.installer.yaml

```diff
@@ -0,0 +1,24 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS0.CRLF.7-3-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: OpenStreetMap.Osm2pgsql
+PackageVersion: 1.7.2
+InstallerLocale: en-US
+Commands:
+- osm2pgsql
+ReleaseDate: 2022-11-10
+Installers:
```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Reply:
The program installs normally, but errors when launched through Windows Explorer: 
![image](https://user-images.githubusercontent.com/114952565/215634285-4040d64a-de98-4792-8686-33027b175f57.png)

When launched through PowerShell, it appears to do nothing. And with the dependency, launching through Windows Explorer also appears to do nothing. Is this the expected behavior for this package?

Reply:
Hi! Yes, that is a expected behavior. osm2psql is a CLI tool. 
Visual C++ Redistributable is not mention as a dependency in the [installation instructions](https://osm2pgsql.org/doc/install.html#installing-on-windows), and it was of already installed on the machine this manifest was tested on.